thali/thalireplicationmanager.js

```diff
@@ -192,11 +206,44 @@ function muxServerBridge(tcpEndpointServerPort) {
   var serverPlex = multiplex({}, function(stream, id) {
     var clientSocket = net.createConnection({port: tcpEndpointServerPort});
     stream.pipe(clientSocket).pipe(stream);
+
+    stream.on('error', function (err) {
+      console.log('Multiplex stream error %s', err);
```

Comment:
I've removed the stream error handling because if that really goes bad then we've got larger problems on our hand.  And no, we don't want to tear that down else we get errors.